### PR TITLE
New UI: snake skin

### PR DESCRIPTION
**Pull request by alexnantel88**

### CODE REVIEW

File: symphony/assets/css/src/installer.less [hunk 1]
```diff
@@ -2,42 +2,101 @@
...
+	padding: 0!important;
 }
 
 form {
-	position: relative;
-	width: 45rem;
-	margin: 0 auto;
-	padding-left: 20rem;
+	box-sizing: border-box;
+	position: relative; margin: 0 auto; padding: @spacing-smallish; max-width: @max-width-small;
```
Comment (nitriques) [inline symphony/assets/css/src/installer.less]: Please use put each declaration on a new line ;)

File: symphony/assets/css/src/symphony.forms.less [hunk 2]
```diff
@@ -165,10 +166,16 @@ button,
...
+	border-radius: @radius-current;
+	display: inline-block; color: @white; font-size: @text-shorter; text-transform: uppercase; line-height: @text-short; margin-left: 1rem; padding: @spacing-smallest @spacing-smallest; border: none; background: @accent; width: auto;
+
+	&:hover {
+		background: @darker-grey;
+	}
 }
 
-.submit input {
+.submit input[type="submit"] {
```
Comment (nitriques) [inline symphony/assets/css/src/symphony.forms.less]: You found some input in a .submit container that was not a submit ?

**Review by nitriques — CHANGES_REQUESTED** [review]

**Comment by nitriques** [comment]
Cool! We might wanna keep the red background around errors ?

**Review by nitriques — APPROVED** [review]

File: symphony/assets/js/src/backend.js [hunk 3]
```diff
@@ -68,6 +68,96 @@
...
+			tableEl: 'table td',
+			dashboard: '#dashboard',
+			dashboardDrawerSelects: '#drawer-dashboard select:not(.disabled)',
+			dashboardDrawerSelectsArrows: '#drawer-dashboard .select2-container .select2-selection--single .select2-selection__arrow',
+			selectArrows: '.select2-container .select2-selection--single .select2-selection__arrow',
+			typeChangerEl: '#custom-toolbar .type-changer a',
+			focusOptionEl: '#custom-toolbar .focus-option a',
+			customToolbar: '#custom-toolbar',
+			dataSourceSource: '#ds-context'
+		};
```
Comment (nitriques) [inline symphony/assets/js/src/backend.js]: Please check how we define "roots" at line 57. Can you remove that big ass object an only add the one or two that you need please ?

File: symphony/assets/js/src/backend.js [hunk 4]
```diff
@@ -68,6 +68,96 @@
...
+
+		/*$(o.header).append('<a href="" id="btn-toggle-header-mobile">'+s.burger+'</a>');
+
+		$(o.header).on('click', o.btnMobileNav, function(){
+			$(o.header).toggleClass('opened');
+
+			return false;
+		});*/
+
+		/* Init - Show Subnav if parent active and highlight Subnav page if the case */
```
Comment (nitriques) [inline symphony/assets/js/src/backend.js]: We try to never commit commented code, it's  bad practice :package:

File: symphony/assets/js/src/backend.js [hunk 5]
```diff
@@ -68,6 +68,96 @@
...
+
+		$(o.navEl + '.active').has('ul').each(function(){
+			var t = $(this);
+			t.addClass('opened');
+			$('ul', t).show();
+
+			$('ul > li a', t).each(function(){
+				var t = $(this);
+				if(window.location.href.indexOf(t.attr('href')) !== -1) t.parent().addClass('active');
+			});
```
Comment (nitriques) [inline symphony/assets/js/src/backend.js]: This should be done in PHP. Check the `__buildNavigation()` method in class.administrationpage.php

**Review by nitriques — CHANGES_REQUESTED** [review]

File: symphony/assets/js/src/backend.js [hunk 6]
```diff
@@ -116,38 +79,15 @@
...
-		$(o.navEl).has('ul').on('click', '> span', function(){
+		$('li', Symphony.Elements.nav).has('ul').on('click', '> span', function(){
 			var t = $(this);
 
 			// Open
 			if(!t.parent().hasClass('opened')) {
-				$(o.navEl + '.opened ul').slideUp(250);
-				$(o.navEl + '.opened').removeClass('opened');
+				$('.opened ul', Symphony.Elements.nav).slideUp(250);
+				$('.opened', Symphony.Elements.nav).removeClass('opened');
```
Comment (nitriques) [inline symphony/assets/js/src/backend.js:103]: Much better don't you think ? ;)

**Review by nitriques — APPROVED** [review]

File: symphony/assets/js/src/backend.js [hunk 7]
```diff
@@ -68,6 +68,36 @@
 
 		// Render view
 		Symphony.View.render();
+
+		/* ////////////////////////////////////////////////////////////////////////////
+		//
+		// Header and Context
+		//
+		/////////////////////////////////////////////////////////////////////////// */
+
```
Comment (nitriques) [inline symphony/assets/js/src/backend.js]: This can be only

```js
// Header and Context
```

**Review by nitriques — CHANGES_REQUESTED** [review]

File: symphony/lib/toolkit/class.administrationpage.php [hunk 8]
```diff
@@ -801,7 +805,8 @@ public function appendNavigation()
 
             if ($this->doesAuthorHaveAccess($n['limit'])) {
                 $xGroup = new XMLElement('li', '', array('role' => 'presentation'));
-                $xGroupLabel = new XMLElement('span', $n['name']);
+                if(is_array($n['children']) && !empty($n['children'])) $xGroupLabel = new XMLElement('span', $n['name'].Widget::SVGIcon('chevron'));
+                else $xGroupLabel = new XMLElement('span', $n['name']);
                 $xGroup->appendChild($xGroupLabel);
```
Comment (nitriques) [inline symphony/lib/toolkit/class.administrationpage.php:949]: Please also use multiple lines for this. And always add curly brakets.

**Review by nitriques — CHANGES_REQUESTED** [review]

**Comment by animaux** [comment]
Thanks for all the great work you guys put into this!

By permission of @nitriques I will try to give some feedback. I will also try to be as objective as possible, but as things go with design stuff there is always a great deal of things that depend more on taste than anything else.

I know this is work in progress, so please excuse me if I go too far `:)`

## Nice

+ Roboto and apparent material design inspired approach
+ Clean and minimal
+ Top-right save button and back-arrow for current section

## UI-design issues

+ Light font weight seems finicky on non-retina devices
+ Fieldset labels in caps are hard to read und get longish
+ header h1 background orb seems unneccessary
+ site titles of medium length already breaks the layout
+ input fields feel inverted, when following the brighter=more-important dogma

## Minor UI-design issues

+ Gray text contrast seems to low (already on @nitriques’ radar)
+ Fieldset legends seem oversized (when in »blue mode«)
+ Fieldset labels in caps are hard to read 

## Possibly a question of subjective taste?

+ Our customers are always very fond of the simplicity of the current UI. The fact that everything important can be found in the main horizontal nav makes it very clean. I do like the fact that the new vertical navigation shows everything at once, but the page feels much more cluttered in L-form.
+ Using lines as dividers: I feel this could lean more towards the concept of structure/layers/areas in the [material design guidelines](https://material.io/guidelines/layout/structure.html).
+ border-radius and padding seems to large on inputs and sections. Maybe we should consider [material design](https://material.io/guidelines/components/text-fields.html#text-fields-input-types) here too?

Since you have been going the Roboto-way I wonder if there aren’t a lot more design-cues from the Material design guidelines we could implement. Even though I don’t even own an android device I strongly feel that the whole visual and ux-approach seems to be a lot more consistent and thought through than f. e. iOS 7+.

**Pull request closed without merging**